### DISCUSSION TITLE
Can you disable boxes around heading and code blocks in Markdown?

### DISCUSSION THREAD

**Discussion by NamamiShanker** [Textualize/rich#1185] [Q&A]
I dont want to print the boxes which are printed around headings and code blocks in Markdown class. Is there any way to disable it?
The picture below for reference.
![query](https://user-images.githubusercontent.com/26577306/115892440-c4cf1780-a474-11eb-9143-0ef981a68f4c.png)

**Comment by willmcgugan** [comment]
Why do you want to? The screenshot you sent me on twitter had the box characters replaced with `?`. This happens when you encoding a str as `ascii` with `error="replace"`.

**Reply by NamamiShanker** [reply]
Ok, I understood. I was using [`simple-term-menu`](https://github.com/IngoMeyer441/simple-term-menu) to display Markdown text in the preview box. I suppose their library doesn't support `UTF-8`, for which I have raised an [issue](https://github.com/IngoMeyer441/simple-term-menu/issues/23). Thanks for helping : )
